projects/ngx-d3timeline/src/lib/axis/resources-axis/resource-axis-tick-renderer.ts

```diff
@@ -12,8 +20,19 @@ export class ResourceAxisTickRenderer implements TickRenderer {
     return tick;
   }
 
-  getTransform(tick: any): number {
-    return this.getBandMidPoint(tick);
+  getTransform(tick: any): string {
+    const axisOffset = this.getBandMidPoint(tick);
+    return this.optionsService.getTranslation(
+      axisOffset,
+      this.orientation === Orientation.Horizontal ? 0 : -10,
```

Comment:
I think there is too much logic here. Would be better to keep `getTranslation` to only depend on a range value, the view and orientation. Use dx and dy to position the tick relative to the translation by adding dx and dy properties to the `Tick` model. I created #79 and #80 for this.

Reply:
see comment below re getTranslation.

Reply:
Thought this through more, I now see a problem in that the position of the tick line and the tick label will differ and have different cx/cy. Most likely for the time axis cx/cy are 0,0 for the tick line, and say 0,-10 for the tick label. Largely overlooked this as I was focusing on the resource axis in this pull request where there is no tick line.

Reply:
Should I leave this as is in this PR and address in the newly created issues?

Reply:
I would say just remove even if it draws incorrectly. This will also keep the PR small.